### ISSUE TITLE
Can't update layout for my taxonomies.

### ISSUE THREAD

**Issue by j-jalving** [statamic/cms#8404]
### Bug description

I've created a categories taxonomy for my blog collection. But when loading the `index` and `show` views, I get the error `View [layout] not found.`. 

The problem is that my layouts are in a "layouts" subfolder, but it seems impossible to change the layout for my taxonomies.

The only workaround that I've found is to create a hidden field on the taxonomy blueprint with a default value of 'layouts/default'. But this does not solve the issue for the taxonomy index page.

### How to reproduce

1. Put your default `layout` in a layouts subfolder.
2. Create a taxonomy and it's views.
3. Open the view in youw browser.

### Environment

```yaml
Environment
Application Name: MyWebsite
Laravel Version: 10.14.1
PHP Version: 8.2.7
Composer Version: 2.5.8
Environment: local
Debug Mode: ENABLED
URL: my-website.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: ses
Queue: sync
Session: file

Localization
Installed: en, nl
LaravelLang\Attributes\Plugin: v2.3.3
LaravelLang\HttpStatuses\Plugin: v3.3.1
LaravelLang\Lang\Plugin: 12.23.0
Protected Locales: nl
Publisher Version: v14.6.4

Statamic
Addons: 7
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: 4.9.2 Solo

Statamic Addons
aerni/livewire-forms: 8.1.0
aerni/social-links: 3.1.0
dryven/faviconator: 1.2.2
gerttimmerman/statamic-zapier: 1.4
jonassiewertsen/statamic-livewire: 2.12.0
rias/statamic-redirect: 3.1.0
statamic/seo-pro: 5.2.0
```

### Installation

Fresh statamic/statamic site via CLI

### Antlers Parser

None

**Comment by duncanmcclean** [comment]
I believe this may be solved by #8372.

**Comment by duncanmcclean** [comment]
I'm going to close this issue as I've realised this is a duplicate of #2003.

There's an open pull request (#8372) which should solve this issue for you.
